demos/interpreter/index.html

```diff
@@ -183,6 +182,7 @@
         if (!ok) {
           // Program complete, no more code to execute.
           document.getElementById('stepButton').disabled = 'disabled';
+          workspace.highlightBlock(null);
```

Comment:
Does this unhighlight all blocks?

If so, the doc should highlightBlock should call out that highlightBlock(null) will unhighlight everything.